gnmi_server/server.go

```diff
@@ -160,7 +163,7 @@ func NewServer(config *Config, opts []grpc.ServerOption) (*Server, error) {
 	if srv.config.EnableTranslibWrite || srv.config.EnableNativeWrite {
 		gnoi_system_pb.RegisterSystemServer(srv.s, srv)
 	}
-	if srv.config.EnableTranslibWrite {		
+	if srv.config.EnableTranslibWrite {
```

Comment:
Lets avoid redundant check.
```
if srv.config.EnableNativeWrite {
       gnoi_system_pb.RegisterSystemServer(srv.s, srv)
}
if srv.config.EnableTranslibWrite {
	gnoi_system_pb.RegisterSystemServer(srv.s, srv)
	spb_gnoi.RegisterSonicServiceServer(srv.s, srv)
}
```

Reply:
@kwangsuk I only corrected the formatting. only extra spaces trailing the line was removed. Anyways I have removed extra space in the new patch.